test/sequential/sequencerStaking.minStake.test.ts

```diff
@@ -0,0 +1,363 @@
+/*
+ *
+ * @group L1RolldownUpdates
+ */
+
+import {
+  ChainName,
+  SequencerStaking,
+} from "../../utils/rollDown/SequencerStaking";
+import { getApi, initApi } from "../../utils/api";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { AssetWallet, User } from "../../utils/User";
+import { Sudo } from "../../utils/sudo";
+import { Assets } from "../../utils/Assets";
+import { GASP_ASSET_ID } from "../../utils/Constants";
+
+import { expectMGAExtrinsicSuDidSuccess } from "../../utils/eventListeners";
+import { BN_ONE, BN_ZERO, signTx } from "gasp-sdk";
+import {
+  createAnUpdateAndCancelIt,
+  L2Update,
+  Rolldown,
+} from "../../utils/rollDown/Rolldown";
+import { BN_TWO } from "@polkadot/util";
+import BN from "bn.js";
+
+const chain: ChainName = "Ethereum";
+let testUser: User;
+let testUser2: User;
+let testUser3: User;
+let testUser4: User;
+beforeAll(async () => {
+  await initApi();
+  await setupApi();
+});
+
+beforeEach(async () => {
+  [testUser, testUser2, testUser3, testUser4] = setupUsers();
+  await Sudo.batchAsSudoFinalized(
+    Assets.mintNative(testUser),
+    Assets.mintNative(testUser2),
+    Assets.mintNative(testUser3),
+    Assets.mintNative(testUser4),
+  );
+  testUser.addAsset(GASP_ASSET_ID);
+  testUser2.addAsset(GASP_ASSET_ID);
+  testUser3.addAsset(GASP_ASSET_ID);
+  testUser4.addAsset(GASP_ASSET_ID);
+  await SequencerStaking.removeAllSequencers();
+});
+
+async function setup3sequencers(
```

Comment:
wouldnt be better to store those helper fns in some utils? but not sure how is the current state, so just askin

Reply:
those 3 sequencers fn is only for this test, but 100% to utilize this whenever we need more. :)